src/vma/util/vma_stats.h

```diff
@@ -200,6 +205,7 @@ typedef struct {
 		n_rx_ready_pkt_count = n_rx_ready_byte_count = n_rx_ready_byte_limit = n_rx_zcopy_pkt_count = n_tx_ready_byte_count = 0;
 		memset(&counters, 0, sizeof(counters));
 		mc_grp_map.reset();
+		ring_user_id_rx = ring_user_id_tx = 0;
```

Comment:
Need to earse ring_alloc_logic_rx/ring_alloc_logic_tx

Reply:
- Fixed the sockopt inheritance issue.
-  How will I know that the logic is user_id to print the ID if I remove ring_user_id_rx/tx ? the ID itself is not enough.